products/error_tracking/backend/max_tools.py

```diff
@@ -205,3 +210,69 @@ async def _arun_impl(self, instructions: str) -> tuple[str, ErrorTrackingIssueIm
             except Exception as e:
                 raise ValueError(f"Failed to generate ErrorTrackingIssueImpactToolOutput: {e}")
         return content, events
+
+
+class ErrorTrackingExplainIssueArgs(BaseModel):
+    """TODO: is this needed?"""
+
+
+class ErrorTrackingExplainIssueOutput(BaseModel):
+    """Structured output for issue explanation"""
+
+    generic_description: str = Field(description="A comprehensive technical explanation of the root cause")
+    specific_problem: str = Field(description="A detailed summary of exactly how the issue occurs")
+    possible_resolutions: list[str] = Field(
+        description="A list of potential solutions or mitigations to the issue", max_length=3
+    )
+
+
+class ErrorTrackingExplainIssueTool(MaxTool):
+    name: str = "error_tracking_explain_issue"
+    description: str = "Given the stack trace and context of an error tracking issue, provide a summary of the problem and potential resolutions."
+    args_schema: type[BaseModel] = ErrorTrackingExplainIssueArgs
+
+    async def _arun_impl(self) -> tuple[str, dict[str, Any]]:
+        validated_context = ErrorTrackingExplainIssueToolContext(**self.context)
+
+        analyzed_issue = await self._analyze_issue(validated_context)
+        user_message = self._format_explanation_for_user(analyzed_issue, validated_context.issue_name)
+
+        return user_message, {}
+
+    async def _analyze_issue(self, context: ErrorTrackingExplainIssueToolContext) -> ErrorTrackingExplainIssueOutput:
+        """Analyze experiment and generate summary."""
+        stacktrace = self.context.get("stacktrace")
+
+        if not stacktrace:
+            raise ValueError(f"No stacktrace provided")
+
+        formatted_prompt = ERROR_TRACKING_EXPLAIN_ISSUE_PROMPT.replace("{{{stacktrace}}}", stacktrace)
+
+        llm = MaxChatOpenAI(
```

Comment:
Add `inject_context=False` to remove the agent personality prompts that we inject by default, which are useless here.